tests/test_statistics.py

```diff
@@ -1,55 +1,119 @@
-from sporkfish.statistics import Statistics
+import chess
+import pytest
+from init_board_helper import board_setup, evaluator, init_board
 
-# TODO: add tests to check searcher is indeed incrementing statistics correctly
+from sporkfish.board.board_factory import Board, BoardFactory, BoardPyChess
+from sporkfish.searcher.move_ordering.move_order_config import (
+    MoveOrderConfig,
+    MoveOrderMode,
+)
+from sporkfish.searcher.searcher_config import SearcherConfig, SearchMode
+from sporkfish.searcher.searcher_factory import SearcherFactory
+from sporkfish.statistics import NodeTypes, PruningTypes, Statistics, TranpositionTable
 
 
 class TestIncrementStatistics:
-    def test_increment_node_visited(self):
+    def test_increment_visited(self):
         S = Statistics()
```

Comment:
Overly extreme nitpick but capital letters are supposed to be for constants, though in a test no big deal